### ISSUE TITLE
TAN hotline overloaded - infected people not getting through and giving up

### ISSUE THREAD

**Issue by vaubaehn** [corona-warn-app/cwa-hotline#14]
## Which hotline
TAN hotline

## What is missing
Enough staff that can pick up calls from users

## Why should it be included
To enable more users to warn their contacts

## Additional information
Hi there,
according to current Google Play Store reviews for CWA, the telephone hotline for TANs (entering TAN to warn risk contacts) is overloaded! People are waiting >30min, many are giving up... Resulting in many people not getting warned about their infection risk.
It looks like that the responsible companies have not been prepared for the higher incidence rates, and too less people are currently on phone to pick up user calls.

Is there anything you can do from your side, e.g., @thomasaugsten to forward a red alert to the responsible people?
Thank you in advance!

cc: @dsarkar @heinezen 

---
17.11.2021

https://play.google.com/store/apps/details?id=de.rki.coronawarnapp&reviewId=gp%3AAOqpTOHDapzxGQivmkPMCqrqxJLvLJKZ1A91kFN8fzCIt-7tMuxv9frQkiMeP_AMrrNB81Ss9L7-6Bmt4J0Sxg

https://play.google.com/store/apps/details?id=de.rki.coronawarnapp&reviewId=gp%3AAOqpTOFoS3n2wK8ZX1wmImRqqvscgu2rH_6wxUj6YsRyLNr99NTtXjkRE_5wYJeQ-FwT1adnb4J-9wIgeTA0KQ

https://play.google.com/store/apps/details?id=de.rki.coronawarnapp&reviewId=gp%3AAOqpTOEIuJr8gmWNj7WC8U9R7FX6JZXkGGGSj-IVScR3mWca5BDeLW6FlRjsVDXWnEi2_YvjcMHNg_gPQhPMMg

18.11.2021

https://play.google.com/store/apps/details?id=de.rki.coronawarnapp&reviewId=gp%3AAOqpTOGybTPRzpd_dydgFFWwxFJzzXMMPM5DLnoGtbHkCN4IktI7Ju9BIqMq8ky1peWD0wFYUFKPDRiFdpw1-w

---
Internal Tracking ID: [EXPOSUREAPP-10939](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10939)

**Comment by vaubaehn** [comment]
Aaaand twitter reports:

https://twitter.com/RA_AndrBach/status/1460881152382836742?s=20

**Comment by Ein-Tim** [comment]
Here's someone complaining that the TAN hotline did not call back: https://twitter.com/fraujen2/status/1459107240565260288?s=21

**Comment by vaubaehn** [comment]
@Ein-Tim
That twitter report is from November 12th. According to https://www.coronawarn.app/en/analysis/ for many months there has been requests for TANs <300 daily. Since beginning of November there is a strong increase of TANs, since end of last week there are remarkable peeks. So the problem may already exist for some days, and now very quickly going worse...

**Comment by vaubaehn** [comment]
Good evening, @daniel-eder 
I just found out that TSI is responsible for the TAN hotline. Are you able to forward the urgent problem reported here to the related parties?
Thank you in advance for your feedback!

**Comment by daniel-eder** [comment]
Hi @vaubaehn, thank you for the report and for the ping. I'm not directly involved with the Hotline, but I'll forward this to our internal stake holders, although i suspect they are aware of the issue (if all operators are on 24/7 with long waiting times for callers that is visible internally as well)

**Comment by vaubaehn** [comment]
Dear @daniel-eder , perfect, thanks a lot. Then we'll most likely see a better response from the call center soon.
Have a relaxing night!

**Comment by Ein-Tim** [comment]
@dsarkar 

Can you relabel this issue to `bug`, please? 

---

Another [Twitter report](https://twitter.com/tommy_bamberg/status/1461632056711168003?s=21): 
"Gestern vier mal bei der Hotline der @coronawarnapp angerufen um mir einen TAN zu besorgen.
Vier mal wurde ich aus der Leitung gekegelt..
Dann melden wir uns halt nicht #Covid_19 #CoronaWarnApp"

**Comment by vaubaehn** [comment]
Here are also two more reports in Google Play Store from today:

https://play.google.com/store/apps/details?id=de.rki.coronawarnapp&reviewId=gp%3AAOqpTOGWNcMZVEJNxVHS7UjnkcqOgWMpOCPR-HEa4QTiYPEOF1Gg_LGUef6CjwpvpaXopn-ukfnGLeO4rEHw9g

https://play.google.com/store/apps/details?id=de.rki.coronawarnapp&reviewId=gp%3AAOqpTOFWALCA6aB73zPxi9lpyFX09TwgxW4zDDfbCexz-70RytZM4MVtQ17SbZvSQMymKsgbG6dskj3Z9c1edw

it's sad that it's still an issue.

And combined with the other issue, that people can't register their test QR codes, it's kind of a catastrophe currently.

**Comment by Ein-Tim** [comment]
Another [Twitter report](https://twitter.com/holzi83/status/1461712902105280514?s=21):
"Ich mag keine Wut-Posts. Der muss aber sein: Ich versuche seit einer Stunde das positive Testergebnis eines Familienmitglieds in die #CoronaWarnApp zu bekommen. Kein QR-Code funktioniert. Bei der Hotline nach 20 Minuten Warteschleife aufgelegt. Projekt ist mMn leider gescheitert."

**Comment by Ein-Tim** [comment]
Twitter reports (this list will be extended over time):

- https://twitter.com/baganabarand/status/1462004533131984897?s=21

**Comment by Ein-Tim** [comment]
@daniel-eder & @thomasaugsten 

Is there any improvement planned here?

**Comment by thomasaugsten** [comment]
Please forward the collection to the corona warn mail of the RKI

**Comment by Ein-Tim** [comment]
@thomasaugsten 

You mean I should send the list of affected Twitter users to coronawarnapp@rki.de?

**Comment by thomasaugsten** [comment]
Yes there have maybe a communication to handle this

**Comment by Ein-Tim** [comment]
@thomasaugsten Okay, done. The hotline is not under influence of TSI/SAP?

**Comment by thomasaugsten** [comment]
The hotline is not related to the GitHub here. This is the reason to communicate the issue directly.

**Comment by MikeMcC399** [comment]
Although the hotline issue is not well suited to being reported here in GitHub, it would be good to know if there are any agreed metrics for the TAN hotline (Service Level Agreement).

Typical call center software will report on such metrics as:
- Number / percentage of lost calls
- Number of answered calls
- Call waiting times (shortest, average, longest)
- Call duration

so I would assume that the management of the TAN hotline is well aware that there is an issue.

There hasn't been any real feedback here so we don't know if there is a technical issue with the hotline or if there are simply insufficient call agents available to handle the current call volume.

**Comment by daniel-eder** [comment]
Any insights into the contractual situation or even the metrics really can only be given by the client, in this case the RKI. They do not have an official presence here, so the best we can do is communicate anything mentioned here internally.
That said, I think it is very much a valid concern to look for feedback on the situation, but I can only refer to the already mentioned e-mail address of the RKI - they might be able to provide more insight into their current view and the planned next steps.

**Comment by Ein-Tim** [comment]
FYI: I contacted the RKI via coronawarnapp@rki.de. They did not yet answer.

**Comment by Ein-Tim** [comment]
Answer from the RKI to my mail:

"[…] Wir arbeiten mit Hochdruck an der Steigerung der Hotline-Kapazitäten, jedoch ist dies aus vielerlei Gründen mit erhöhtem Aufwand verbunden. Wir bitten die Verzögerung zu entschuldigen und empfehlen, es zu einem anderen Zeitpunkt erneut zu versuchen. […]

I really hope we will see an improvement soon...

**Comment by dsarkar** [comment]
---
Internal Tracking ID: [EXPOSUREAPP-10939](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10939)

**Comment by Ein-Tim** [comment]
There are less Twitter reports on this, it seems like the situation has improved (or less users are getting infected and want to request a TAN).

**Comment by Ein-Tim** [comment]
It seems like the situation got worse again.

- https://twitter.com/kirshteinkoos/status/1484209260754083840?s=20 (waiting ca. 25 min. before hanging up)
- https://twitter.com/ChristianGunkel/status/1483916588495646720?s=20 (waiting over one hour in the hotline before hanging up)
- https://twitter.com/derfuchsifuchs/status/1484599448323960837?s=20 (can't come through since two days)

**Comment by vaubaehn** [comment]
Yes, there were also again few reports in Google Play Store.

**Comment by Ein-Tim** [comment]
@vaubaehn 

Actually, the working hours have been reduced from 24/7 to 07:00h to 20:00h.

**Comment by vaubaehn** [comment]
@Ein-Tim 
Maybe you could write to RKI once more? ❤️ 
I don't know who's responsible for this planning, but my gut feeling is, it's connected with funding... Maybe you could ping Karl Lauterbach on Twitter?

To be honest, this kind of misplanning and obviously forseeable faults affect my motivation to engage into the project volunteerly rather much in a negative way.

**Comment by vaubaehn** [comment]
> @vaubaehn
> 
> Actually, the working hours have been reduced from 24/7 to 07:00h to 20:00h.

I wasn't aware :(

**Comment by vaubaehn** [comment]
@Ein-Tim 


> I will write to the RKI via Mail again.

Thanks a lot.
> I can also ping Karl Lauterbach, but I doubt that I get an answer.

I also wouldn't expect an answer, but something like
"Es mehren sich Berichte, dass Covid-Infizierte bei der TAN Hotline nicht mehr durchkommen, um Codes für die Übermittlung einer Warnung über die Corona-Warn-App zu erhalten. Zudem wurden zu Beginn der Omikronwelle die Geschäftszeiten der Hotline verkürzt. Könnten Sie bitte schauen, was da los ist und ggf. (finanziell) nachsteuern? Vielen Dank!"
and putting RKI, Corona-Warn and maybe some media in cc. With your reputation it may have an effect, even without answer.
I don't have twitter unfortunately, and of I opened an account just for this, I would miss the reputation...

> > To be honest, this kind of misplanning and obviously forseeable faults affect my motivation to engage into the project volunteerly rather much in a negative way.
> 
> I would guess that there was some kind of internal analysis when the fewest calls reached the hotline and probably this was the time between 20:00h and 07:00h (however, I wonder why the technical hotline is still reachable from 07:00h till 22:00h).

Yes, that's probably the case. Most likely the intention was to safe some costs... But this is saving money at the wrong place as it's undetermining some crucial basic concepts of the whole project. To get a positive public perception and to be taken serious by the public, such a hotline needs to be reachable 24/7. Saving some 10.000 Euros is not worth damaging the whole project in these basic necessities.

@dsarkar additionally, the FAQ states that BOTH hotlines are **closed** during national holidays - could you please cross-check whether this is true? Thank you!
https://www.coronawarn.app/en/faq/#international_phone_numbers

**Comment by Ein-Tim** [comment]
@vaubaehn 

See my PM in Slack.

> @dsarkar additionally, the FAQ states that BOTH hotlines are closed during national holidays - could you please cross-check whether this is true? Thank you!

This should definitely be cross checked as the app itself does not say so.

**Comment by vaubaehn** [comment]
After internal discussion, @Ein-Tim published this tweet:
https://twitter.com/EinTim2/status/1484822606842900481?s=20

Thank you, @Ein-Tim ! ❤️

**Comment by dsarkar** [comment]
@Ein-Tim @MikeMcC399  @vaubaehn Thanks for pinging me. 
@MikeMcC399 or @Ein-Tim Would you provide a PR and remove "(except on German national holidays)." from the TAN hotline? Let me know, please, thanks!

**Comment by Ein-Tim** [comment]
@dsarkar 

Yes I will provide a PR.

FYI @MikeMcC399 & @vaubaehn

**Comment by MikeMcC399** [comment]
@Ein-Tim 
Thanks for picking this up! The next German public holiday is not until Apr 15, 2022, so you have plenty of time to get the information fixed!

**Comment by Ein-Tim** [comment]
PR https://github.com/corona-warn-app/cwa-website/pull/2368 opened to fix the wrong information reg. the reachability of the TAN hotline on German national holidays in the FAQ.

**Comment by dsarkar** [comment]
@Ein-Tim thanks also for your comment https://github.com/corona-warn-app/cwa-hotline/issues/14#issuecomment-1018775419

We will forward this to the corresponding entity.

**Comment by Ein-Tim** [comment]
Answer from the RKI to my second mail: 

> Um dem in den letzten Tage stark erhöhten Anruferaufkommen während der Tagesstunden gerecht zu werden, wurden die Servicezeiten der Verifikationshotline umgestellt - wie Sie zu Recht vermuten, um die Personalkapazität zu erhöhen. Leider benötigt auch diese Umstellung etwas Zeit, bitte seien Sie versichert, dass wir auch daran mit Hochdruck arbeiten! Bis zu einer reibungslosen Umsetzung können wir Sie nur um Ihre Geduld bitten.

**Comment by vaubaehn** [comment]
@Ein-Tim 
Thanks a lot.
But I still don't get how this could happen as this situation was completely foreseeable quite some time ago. At least measures are taken (again).

**Comment by Ein-Tim** [comment]
I still update https://github.com/corona-warn-app/cwa-hotline/issues/14#issuecomment-974631128. 

Meanwhile, more and more users are reporting that they get through after some time, but they are not called back.

**Comment by dsarkar** [comment]
@Ein-Tim Thanks for letting us know. Will be forwarded.

**Comment by vaubaehn** [comment]
According to some tweets you collected @Ein-Tim , hotline staff seems exhausted thus reacting unfriendly. Sounds like a lose-lose-situation for everyone. (related: https://github.com/corona-warn-app/cwa-hotline/issues/15)

**Comment by Ein-Tim** [comment]
There are multiple users on Twitter reporting that through the huge load the service quality decreased. 
[This](https://twitter.com/anni_we_/status/1487407986700136451?s=21) Tweet e.g. says:

> Kann ich nachvollziehen! Wenn ich dann aber jemanden erreiche, die Person völlig genervt meine T-Nr. falsch aufschreibt und auflegt während ich die Nummer korrigieren will und ich jetzt wieder stundenlang in der Hotline hänge fehlt mir sämtliches Verständnis

This is a loose-loose situation for everyone, the hotline staff is exhausted and thus the quality of the service is bad and the user is also unhappy because they had to wait for a long time anf then there is someone who is really stressed and does rush through the process.

There is only one way to improve the situation: Increase the hotline team.

**Comment by vaubaehn** [comment]
I'm getting a bit frightened to see how @Ein-Tim 's list of collected _sample_ complaints extends.

**Comment by Ein-Tim** [comment]
Two twitter users are reporting that they only had to wait 10-15 min. This is acceptable. Maybe the situation improved? 
https://twitter.com/DnlMndc/status/1488142266275405825?s=20&t=FvrX7Lm1Bqy7dhlhwrt3lQ

**Comment by xlr5** [comment]
Today I waited >45 minutes until I got someone who gave me a TAN

**Comment by dsarkar** [comment]
@Ein-Tim @xlr5 Will forward your feedback
Thanks.

**Comment by Ein-Tim** [comment]
Judging from Twitter reports the situation today is really bad, waiting times between 45 Min & 1.15h.

**Comment by vaubaehn** [comment]
According to Google Play Store, someone tried for 2hrs yesterday, wrote he let it ring in one single call... Don't know whether the waiting line keeps callers actually such long in the line...
Others reported to give up after 30/45 mins...

Doesn't look like it was possible to organize a real improvement during the last 3 weeks.

**Comment by Ein-Tim** [comment]
Meanwhile Twitter users are reporting that the waiting times are between 15-30 min. This is acceptable, but most users still think that it is too long.

**Comment by ndegendogo** [comment]
Tbh, I would give up after ~ 5 minutes....

**Comment by vaubaehn** [comment]
Currently, the incidence rates in Germany decrease much slower than compared to other countries some weeks ago.
There is a chance, that we might reach a plateau soon. According to studies of TU Berlin, we should take the risk of again increasing incidence rates into account, due to the spread of Omicron subtype B2:
https://depositonce.tu-berlin.de/bitstream/11303/16461/4/2022-02-23_MODUS-COVID_Bericht.pdf

The incidence rates could reach a level compared to some weeks ago, or in worst case exceed past incidence rates by factor 2.5

This would mean: again many people calling the TAN hotline.
Is the hotline prepared for such a scenario in the next weeks?

@ndegendogo 
> Tbh, I would give up after ~ 5 minutes....

Would it change anything, if they had nice music or quiz played for the waiting time?

**Comment by Ein-Tim** [comment]
Twitter reports have changed from: "I have to wait more than 45 min in the hotline" to "I have to call the hotline twice because the staff does not call back". 

**Questions:**
- Does the staff give a time frame when they will call back?
- Does the staff suggest to call again if there is no call back?

**Comment by ndegendogo** [comment]
> Would it change anything, if they had nice music or quiz played for the waiting time?

I think, on the first attempt: no. 
After a positive test result I need to organize so much else. Notify my friends, family and collegues. Find out how to self-isolate best, and if/how I can continue to work. Organize my daily shoppings. Find out who else needs to be notified and what the current and local regulations are .....

Maybe later when all that is settled, I find time for a second try. **And this time with more patience.**

**Comment by dsarkar** [comment]
Good morning @Ein-Tim, 

Thanks for reporting.

> Twitter reports have changed from: "I have to wait more than 45 min in the hotline" to "I have to call the hotline twice because the staff does not call back & give me the TAN".

Do you have some example links from Twitter?

**Comment by Ein-Tim** [comment]
@dsarkar 

Currently not, I'll search, but no guarantee I'll find them 😅

**Comment by vaubaehn** [comment]
Unfortunately, I was wrong im my comment https://github.com/corona-warn-app/cwa-hotline/issues/14#issuecomment-1059333328 four days ago:

> [...] There is a chance, that we might reach...
> ... a plateau...

There is no plateau, the incidences are raising directly.

> ...soon.

It's not soon, but directly happening.

Does anyone know, whether the hotline is prepared to handle an increasing amount of calls in the next weeks?

**Comment by Ein-Tim** [comment]
> Does anyone know, whether the hotline is prepared to handle an increasing amount of calls in the next weeks?

Judging from Twitter reports: No, but it won't be as bad as it was before. I assume max. waiting times of 45 min instead of 2h.

But for sure this is only a prediction from my POV and the truth could be something other.

**Comment by vaubaehn** [comment]
There are many reports in Google Play Store again, that address all the above listed problems:
- waiting times exceeding one hour and more again
- no call backs in many cases
- unfriendly staff (e.g., refusing to assist in situations where a proxy is reporting the infection as the infected person is too old/weak to handle it herself - just closing the line without proper explanation).

With the highest respect to all people involved trying their best, but this is ridicoulus and this situation is unworthy of the project and those affected. In my humble opinion, there should be consequences for the people responsible.

@daniel-eder

**Comment by Ein-Tim** [comment]
@vaubaehn


Judging from Twitter reports, I can only confirm this.

**Comment by vaubaehn** [comment]
> I'll tweet about this today or tomorrow.

Thank's @Ein-Tim , it's so sad that it's necessary.

**Comment by daniel-eder** [comment]
@vaubaehn I raised this issue again internally. Additionally I would like to refer to my comment above ( https://github.com/corona-warn-app/cwa-hotline/issues/14#issuecomment-974872566 ) as the RKI is the decision maker regarding hotline capacity

**Comment by vaubaehn** [comment]
@daniel-eder 
Thanks again for your support. In general it looks like that RKI is aware of the situation but does not know how to change it.
In this case, handling it from the expert's side (like you did by raising it internally again) is probably the most promising way.
But I do agree: Responsible is RKI and the contracted (sub-)companies.

Anyway, thanks again, and I hope you're well in these difficult times.

**Comment by Ein-Tim** [comment]
[This](https://twitter.com/ja_ibalm/status/1501990874242158610?s=21) Twitter report says:

> Bei mir war die Wartezeit heute etwas mehr als 26 Minuten. Aber die Mitarbeiterin muss ich loben: schnell und effektiv! Gesamte Gesprächsdauer inklusive Rückruf und meiner Fragen <3 Minuten 👍

Very strange.

**Comment by Ein-Tim** [comment]
If you yourself ever need a TAN: Call the hotline as early as possible, e.g. at 07:05h.
Twitter users are reporting that this is the least crowded time, which makes sense.

**Comment by ndegendogo** [comment]
> If you yourself ever need a TAN: Call the hotline as early as possible, e.g. at 07:05h

@Ein-Tim thanks - such hints are useful (although, if everybody follows this advice it will be useless again 😬😬😬)

*All experts predict that everybody will be affected some time.*
I cannot expect to be the exception to that rule. And as soon as I will get a positive result, I will indeed need a TAN, for my secondary phone, as cwa integration covers at most one device ...

**Comment by vaubaehn** [comment]
![Screenshot_2022-03-16-09-02-52](https://user-images.githubusercontent.com/68376758/158618332-a8eaa87e-8ce0-47c7-832c-5ed4c4a57f07.png)

**Comment by vaubaehn** [comment]
I think it should be enough now.
See also https://github.com/corona-warn-app/cwa-app-android/issues/4815#issuecomment-1069196770

Is there a way to increase the pressure on RKI/BMG?
@Ein-Tim could you ping Karl Lauterbach, BMG and RKI on Twitter again?
@NicoErnst is this subject something that you could report about?

**Comment by Ein-Tim** [comment]
@vaubaehn I will first write to the mail address again, specifically asking if there are plans to increase capacity.

Please, could you write me a DM via Slack & we discuss the content of the mail?

**Comment by Ein-Tim** [comment]
@vaubaehn 

I'll start drafting a text later on and will send it to you so that you can take a look.
No hurries (-:

**Comment by Ein-Tim** [comment]
@vaubaehn FYI: https://twitter.com/eddieoppermann/status/1504360984097525764?s=21
"fällt manchmal schwer: über 30 Minuten in der Warteschleife, um die TAN für den positiven PCR zu bekommen - für SO VIELE MILLIONEN so wenig Vernetzung mit den Testzentren und SO ein altmodisches  Verfahren 😭 Aber  Chapeau  für  den  freundlichen  Mitarbeiter  in  der  „Hot“line."

**Comment by dsarkar** [comment]
@vaubaehn @Ein-Tim Thanks for the info, will be forwarded.

**Comment by Ein-Tim** [comment]
I have to admit that this is a very frustrating issue for the community. An easy to solve issue (increase capacity of the hotline) remains unsolved for ca. 6 months. While you certainly know what is the blocker here, we, the community, is left in the dark. This is really not encouraging to continue to contribute, at least to this issue. That's why I  I decided to stop updating https://github.com/corona-warn-app/cwa-hotline/issues/14#issuecomment-974631128, as the problem is clear and the situation remains unchanged, for reasons which are not made public in this issue.

**Comment by Ein-Tim** [comment]
I strongly suggest (and will open a wishlist item for this soon) to think about alternatives for the hotline, e.g.:
- Online Chat
- Online form

AFAIC It was decided to use a hotline, among other things, because a call agent can asses the situation and tell how the other person behaves. However, with agents rushing though the process in less than 3 minutes, this is obviously not the case anymore.
Also, it seems like you loose more real warnings with the hotline approach than "fake" warnings are prevented by this.

**Comment by vaubaehn** [comment]
Replacing the hotline with an online chat or online form is likely not a good idea because people with handicaps using chats/forms (like old agers, unexperienced people, ...) will be overwhelmed by this way of TAN retrieval. But chats/forms as an addition/complementation to the Hotline could be a good idea to reduce the load on the call center agents in general. But some questions are left open:
- will there be an additional funding for that? As obviously there is a bad funding for the hotline currently, that seems to be unlikely
- will there be enough staff to answer in chats/on forms? This is dependent on funding
- how long will it take to implement? My guess is: some weeks at least. Probably we're then already out of the 6th wave. But could be good to prepare for the autumn.
- are chats/forms feasible? Agents on the hotline are flexibel to assess/verify the situation for a TAN request in their dialog. A chat would probably result in the same. But can the same be achieved with a form?

> Also, it seems like you loose more real warnings with the hotline approach than "fake" warnings are prevented by this.

I would copy this.

**Comment by Ein-Tim** [comment]
> Replacing the hotline with an online chat or online form is likely not a good idea because people with handicaps using chats/forms (like old agers, unexperienced people, ...) will be overwhelmed by this way of TAN retrieval.

I get your point, but a hotline is also not a good approach if (what often is the case for elderly people) hearing is damaged, etc. 

Without additional funding (if this is the blocker here) I only see two ways: Shut down the hotline and go a different way or leave everything as-is.

> Replacing the hotline with an online chat or online form is likely not a good idea because people with handicaps using chats/forms (like old agers, unexperienced people, ...) will be overwhelmed by this way of TAN retrieval.

Agreed

> are chats/forms feasible? Agents on the hotline are flexibel to assess/verify the situation for a TAN request in their dialog. A chat would probably result in the same. But can the same be achieved with a form?

The form was only an idea, I rather think a chat would be the best, maybe it could even be implemented right into the app.
It would take work from the agents, as the first steps could be performed automatically:
- Upload a picture of the "Laborbefund"
- Agree to the Terms & Conditions, accept the data privacy notice
- Maybe answer some security questions (when was the test carried out, where, etc.)

Only then an agent would have to take a look at this and quickly make sure that everything seems plausible. Afterwards the agent issues a TAN and informs the user that all data is deleted during the next 24h (can also be done automatically).

But we probably won't ever see this, so...

**Comment by Ein-Tim** [comment]
@dsarkar 

Regarding https://github.com/corona-warn-app/cwa-hotline/issues/14#issuecomment-1060360866 see e.g.:

https://twitter.com/eintim2/status/1509617797231427589?s=21&t=jVnkzN7hyLcAgzqMSbi5Gg

**Comment by Ein-Tim** [comment]
Netzpolitik.org wrote an entire article about this hotline mess: https://netzpolitik.org/2022/cwa-tan-hotline-ewige-wartezeiten-bei-der-corona-warn-app-keine-warnung-unter-dieser-nummer/

**Comment by dsarkar** [comment]
@Ein-Tim Thanks for sharing the link here. Also, your Twitter links are forwarded internally.

**Comment by Ein-Tim** [comment]
@dsarkar Just to add it here, although I'm unsure how useful it is, another report "The hotline did not call back" from today: https://twitter.com/SchwebelR/status/1511760533736808452?s=20&t=Ub-Oh7YwwEJtPQ0UVYhRtg

**Comment by vaubaehn** [comment]
Also, via Slack @Ein-Tim pointed me to that article (thanks!), picking up our subject:
https://www.mdr.de/nachrichten/deutschland/corona-newsletter-mittwoch-ja-nein-jein-egal-lauterbach-100.html

**Comment by dsarkar** [comment]
@Ein-Tim @vaubaehn Thanks for sharing these links.

**Comment by dsarkar** [comment]
Forwarded

**Comment by Ein-Tim** [comment]
The situation has improved, no more reports on Twitter. I vote for continuing to monitor the situation and then close this issue. 
This could get critical again in autumn, when COVID-19 cases may be rising again.

**Comment by dsarkar** [comment]
@Ein-Tim Thank you very much for your feedback!

**Comment by Ein-Tim** [comment]
Just chatted with a PCR positive tested person on Twitter, he got a TAN at the hotline and he didn't have to wait longer than 15 seconds in the waiting line.

This really seems fixed.

**Comment by daniel-eder** [comment]
@Ein-Tim I do not yet have information for yesterday. It might be something benign (one day with more traffic than expected) - if the issue persists, however, please give me another ping if possible, so I can forward the information to my colleagues again.

**Comment by Ein-Tim** [comment]
🚨

One more report on this:
- https://twitter.com/PatrickAFelber/status/1534968943558938638?s=20&t=oZtrh80J6gvGwlLnNlz1IA

FYI @daniel-eder

**Comment by biophysicslab** [comment]
Problem is not fixed. Own experience: 
- QR Code from test center scanned by CWA, but result was transferred via Email and does not show up in CWA. 
- Hotline call: 40 min wait yesterday, 30 min today. Giving up.

**Comment by biophysicslab** [comment]
Problem is not fixed. Own experience:

- QR Code from test center scanned by CWA, but result was transferred via Email and does not show up in CWA.
- Hotline call: 40 min wait yesterday, 30 min today. Giving up.

fyi @daniel-eder , @Ein-Tim

**Comment by daniel-eder** [comment]
Thanks @Ein-Tim @biophysicslab. Seems something more is going on - I forwarded your feedback and the collected tweets to the colleagues working with the Hotline

**Comment by Ein-Tim** [comment]
Informationen from one of my sources:

- TAN hotline can currently handle ~ 1000 calls per day
- There are currently 3500 people calling the hotline per day - so already now only ~ 1/3 of incoming calls are processed
- The plan is not to extend capacity but lower capacity to be able to handle 500 calls per day

FYI @vaubaehn & @MikeMcC399

I will comment my opinion on this soon, but currently I'm on vacation and I want to enjoy every minute of it :D

**Comment by thomasaugsten** [comment]
It would be good to have the test labs which are not providing their results to the CWA

**Comment by Ein-Tim** [comment]
@thomasaugsten We agree on this point, but we can't force them. 

Maybe it makes sense to just shut down the hotline, else this is just a "Trauerspiel" - more cases but less capacity on the hotline. Makes no sense.

**Comment by thomasaugsten** [comment]
I have no influence on the hotline part I can only help to provide test via CWA
Maybe it is a technical and not motivation issue

**Comment by ndegendogo** [comment]
@thomasaugsten I faintly remember that integration activities for test providers with cwa were stopped - is this no longer true?

**Comment by Ein-Tim** [comment]
@ndegendogo We don't know if this applies only to RATs or also to PCRs, see https://github.com/corona-warn-app/cwa-documentation/issues/888

**Comment by MikeMcC399** [comment]
@ndegendogo
> I faintly remember that integration activities for test providers / test centers with cwa infrastructure were stopped

Your memory serves you correctly!

https://github.com/corona-warn-app/cwa-quicktest-onboarding/wiki states "Anmeldungen zur API-Anbindung an die Corona-Warn-App (sind) nicht mehr möglich. Auch eine Anbindung über das CWA Schnelltestportal ist nicht mehr möglich."

They refer to [changes in the TestV](https://www.buzer.de/gesetz/14991/al164208-0.htm) starting on March 31, 2022 where the connection to CWA was no longer mandated for § 4a Bürgertestung:

![image](https://user-images.githubusercontent.com/66998419/174604142-864f16a4-9e0f-43db-a222-13ed8edad382.png)

In any case the regulation behind the national test strategy expires on June 30, 2022 and no replacement has been announced. See [TestV](https://www.gesetze-im-internet.de/coronatestv_2021-10/) "Verordnung zum Anspruch auf Testung in Bezug auf einen direkten Erregernachweis des Coronavirus SARS-CoV-2 (Coronavirus-Testverordnung - TestV)", so currently there is no legal basis for related future planning after end of June 2022.

**Comment by Ein-Tim** [comment]
Again @MikeMcC399, are you certain that the registration stop applies to both, RATs and PCRs? As the hotline is only for PCRs, it would be irrelevant if new RAT stations can't be connected to the system.

**Comment by Ein-Tim** [comment]
Updated information from one of my sources:
Hotline capacity will be increased to be able to handle 2000 calls per day during the next weeks.

**Comment by Meyblaubaer** [comment]
Today trying on different times, waiting More than 30 min per Call, giving up.

Sorry but this is Not okay.

Create a E-Mail service where i can send my PCR-Result and obtain a TAN.

**Comment by Ein-Tim** [comment]
A [user on Twitter](https://twitter.com/misseskiki/status/1547059731885445120?s=21&t=x9cexh8uqXfzrHkodl9udg) draw a picture for you: 
![image](https://user-images.githubusercontent.com/67682506/178658245-48f13a91-21c5-4d64-8dac-df7e78d71300.jpeg)

**Comment by dsarkar** [comment]
@Ein-Tim Thanks for forwarding this.

**Comment by dsarkar** [comment]
Hi @benjohnde, we appreciate your feedback, and we are aware of the TAN hotline problematics, as well the integration of the labs into the CWA environment in order to get a CWA test QR code. All these reports are forwarded to the corresponding entities, so that action can be taken.

We have a few questions regarding the testing:
1. Your test  QR code could not be scanned by the CWA, i.e. the test was not registered in the CWA?
2. Therefore, you did not receive the test result via the app?
3. When you registered for a test, were you offered the option to obtain a CWA QR code? 
4. If so, did you "tick that box", but you never received a CWA QR code?
5. Or, is the  test site not connected to the CWA infrastructure, did they tell you they are not offering the result via the CWA?
6. When registering for a test (on site or online?) was there information at all regarding the CWA?

Thanks. Best wishes, DS

---
Corona-Warn-App Open Source Team

**Comment by ndegendogo** [comment]
Ok, so most labs support the cwa interface.
And, as we know, the doctors support it out-of-the-box with the famous KV form.
Only: how many of them are aware of this, and of the procedure, and follow it correctly?

In autumn 2021 I had a red warning, and I wanted a test. So they sent me to my doctor; and the doctor proudly handed me a QR code, issued by the lab, so I can get my result directly from the lab *(and, of course, this was a process without cwa ...)*
When I asked about a cwa-compatible code, they were blank ... So I asked if I can have a copy of the Überweisungsschein to the lab ... they printed it for me, and its QR code worked well ...
**but everybody who had not explicitely requested that printout would then need the TAN from the hotline ...**

**Comment by Ein-Tim** [comment]
FYI: It's starting again:
- https://twitter.com/sufloese/status/1578403687910834181?s=46&t=vWpNf0CCkMzTdvwXFbKSBw
- https://twitter.com/eva_held/status/1578378231622901762?s=46&t=vWpNf0CCkMzTdvwXFbKSBw

I honestly neither have the time nor do I feel a desire to pressure the project partners & stakeholders to assure that their hotline works. They should have understood this by now and if they don't want to change anything, it's not my problem.

**Comment by Ralfeins** [comment]
> Updated information from one of my sources: Hotline capacity will be increased to be able to handle 2000 calls per day during the next weeks.

Thanks for this info @Ein-Tim. Was ever thought about to visualize the capacity of to be handled calls in order to see misconnect to rising corona cases, and act swiftly?

**Comment by Ralfeins** [comment]
Is it possible to share link to currently in place process logic on how positive test result (PCR positive) come back via various ways into the infected person's CWA? Who finally sets real-life process into motion after the laid out business process? BMG, RKI, SAP, other party? 

I know quite a bunch of questions, but my work has to  do with similar, though totally different field, challenges.

**Comment by MikeMcC399** [comment]
@vaubaehn 

With the release of CWA 3.0 it is no longer necessary to use the TAN hotline, and this will be closed down on Jan 31, 2023.

See the blog post [Version 3.0: CWA warnings now possible after positive self-test](https://www.coronawarn.app/en/blog/2023-01-18-cwa-3-0/ ) for more details.

**Comment by MikeMcC399** [comment]
@vaubaehn

This issue is no longer relevant and can be closed, since the TAN hotline closed down today. See [README](https://github.com/corona-warn-app/cwa-hotline/blob/master/README.md).
